src/components/Button/index.js

```diff
@@ -161,94 +161,128 @@ const defaultProps = {
     forwardedRef: undefined,
 };
 
-class Button extends Component {
-    constructor(props) {
-        super(props);
-
-        this.renderContent = this.renderContent.bind(this);
-    }
-
-    componentDidMount() {
-        if (!this.props.pressOnEnter) {
-            return;
+function Button({
+    allowBubble,
+    text,
+    shouldShowRightIcon,
+
+    icon,
+    iconRight,
+    iconFill,
+    iconStyles,
+    iconRightStyles,
+
+    small,
+    large,
+    medium,
+
+    isLoading,
+    isDisabled,
+
+    onPress,
+    onLongPress,
+    onPressIn,
+    onPressOut,
+    onMouseDown,
+
+    pressOnEnter,
+    enterKeyEventListenerPriority,
+
+    style,
+    innerStyles,
+    textStyles,
+
+    shouldUseDefaultHover,
+    success,
+    danger,
+    children,
+
+    shouldRemoveRightBorderRadius,
+    shouldRemoveLeftBorderRadius,
+    shouldEnableHapticFeedback,
+
```

Comment:
those extra spaces are for some purpose?

Reply:
Since there are too many props I wanted to group some of them